### ISSUE TITLE
Not up to date with the latest AWSSDK

### ISSUE THREAD

**Issue by jasonycw** [aws/aws-logging-dotnet#159]
## Description
AWSSDK Package update conflict with `AWS.Logger.SeriLog`

## Reproduction Steps
Update `AWS.Logger.SeriLog` with other AWSSDK packages together

### Logs
```
Version conflict detected for AWSSDK.Core. Install/reference AWSSDK.Core 3.7.0.6 directly to project My.Project to resolve this issue. 
 My.Project -> AWSSDK.S3 3.7.0.6 -> AWSSDK.Core (>= 3.7.0.6 && < 3.8.0) 
 My.Project -> AWS.Logger.SeriLog 2.1.0 -> AWS.Logger.Core 2.1.0 -> AWSSDK.CloudWatchLogs 3.5.0.24 -> AWSSDK.Core (>= 3.5.1.22 && < 3.6.0).
```

### Environment

* Build Version: AWS.Logger.SeriLog 2.1.0
* OS Info: Windows 10
* Build Environment: Visual Studio
* Targeted .NET Platform:  .NET Core 3.1

## Resolution
- [ ] :wave: I can/would-like-to implement a fix for this problem myself

---

This is a :bug: bug-report

**Comment by ashishdhingra** [comment]
Duplicate of https://github.com/aws/aws-logging-dotnet/issues/160. PR https://github.com/aws/aws-logging-dotnet/pull/161 submitted for fix.

**Comment by ashovlin** [comment]
Just released v3.0.0 of https://www.nuget.org/packages/AWS.Logger.SeriLog/3.0.0 which should work alongside AWS SDK v3.7 references.
